.github/workflows/automaton.yml

```diff
@@ -0,0 +1,155 @@
+# Automaton, a beepsky supplement for checking runtime violations in maps
+# Based on Turdis by Yogstation
+
+name: Automaton
+
+on:
+  pull_request:
+    branches: master
+  push:
+
+jobs:
+
+  runtime:
+    name: Runtime Checker
+    runs-on: ubuntu-18.04
+    if: "!contains(github.event.head_commit.message, 'skip ci') && !((github.event_name == 'push' && github.repository == 'goonstation/goonstation') || (github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == 'goonstation/goonstation'))"
+    strategy:
+      fail-fast: false
+      matrix:
+        map: ["COGMAP", "COGMAP2", "CLARION", "DESTINY", "DONUT2", "DONUT3", "HORIZON", "LINEMAP", "MUSHROOM", "TRUNKMAP", "OSHAN", "SAMEDI", "PAMGOC", "KONDARU", "FLEET", "ICARUS", "DENSITY", "ATLAS", "MANTA", "WRESTLEMAP", "SPACE_PREFAB", "UNDERWATER_PREFAB", "FULL"]
```

Comment:
A lot of these maps aren't necessary.

Reply:
I included every map that is playe-pickable in `code/map.dm`, includding maps that show up only during ass jam. I figured if a map *could* be chosen during a round on the public server, it should be checked. Do you think I should remove the ass jam-only maps?

Reply:
Maybe it's just time to remove some of these maps from source. Not sure exactly.